File: java/strings/StringTrimming.java
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

class StringTrimming {
  public static void main(String[] args) {
    String sententia = "   Sapere aude.   ";
    System.out.println(String.format("sententia = <%s>\n", sententia));

    String trimmed_sententia = sententia.trim();
    System.out.println(String.format("trimmed sententia = <%s>\n", trimmed_sententia));
  }
}

File: c#/strings/StringTrimming.cs
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

class StringTrimming {
  public static void Main(string[] args) {
    string sententia = "   Sapere aude.   ";
    System.Console.WriteLine(string.Format("sententia = <{0}>\n", sententia));

    string trimmed_sententia = sententia.Trim();
    System.Console.WriteLine(string.Format("trimmed sententia = <{0}>\n", trimmed_sententia));
  }
}
